platforms/tktrex/backend/scripts/build-indexes.js

```diff
@@ -38,3 +40,6 @@ ret = db.htmls.createIndex({ metadataId: -1 });
 checkret('htmls metadataId', ret);
 ret = db.htmls.createIndex({ processed: 1 });
 checkret('htmls processed', ret);
+
+ret = db.emails.createIndex({ email: 1, reason: 1 }, { unique: true });
+checkret('emails composed email+reason');
```

Comment:
Can we also delete the `subid`-field related code?

Reply:
removed entirely the tiktok related `registerEmail` code (`listEmail` still present) the yt code is without `subId`. both of them should be unify. the issue is that we should manage these email accordingly to the email provider we mighe use to send newsletter. which at the moment we haven't started yet and the provider is not selected.